### DISCUSSION TITLE
How can I get Co-authored-by badge on git.

### DISCUSSION THREAD

**Discussion by PT-3Y** [community/community#67360] [Discussions]
### Select Topic Area

Question

### Body

I am not sure how to get Co-authored badge on git.
I tried several times but I couldn't get it yet.

Please guide me.

**Comment by melvincwng** [comment]
Hey @letsbuildwebsite, this [link](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors) should provide information on how to create a commit with multiple authors, thus allowing you to unlock the 'Pair Extraordinaire' achievement badge.

Follow the step by step instructions on the link above to unlock that achievement badge which you are aiming for.

Hope this clarifies and answers your question 😃

**Reply by PT-3Y** [reply]
Thanks  for your answering @melvincwng.

**Comment by MkDev11** [comment]
Please send an invitation to your collaborators.

Please work on new branch.

And make a change.

Commit the change with ``git commit -m  "commit text" -m "Co-authored-by: username <usermail>"``

And push the commit.

And merge this branch into main branch.

It should make co-work badge.

**Reply by PT-3Y** [reply]
Awesome! 
Thank you @MkDev11  
It worked well for me.
